### PR TITLE
[DC-1100] Create get_date_columns utility for deactivation retraction

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-1100] Update query and fix tests 
* [DC-1100] Fix tests in related cleaning rule

[DC-1100]: https://precisionmedicineinitiative.atlassian.net/browse/DC-1100
[DC-1100]: https://precisionmedicineinitiative.atlassian.net/browse/DC-1100

### CODE REVIEW

File: data_steward/retraction/retract_deactivated_pids.py [hunk 1]
```diff
@@ -52,6 +33,37 @@ def get_client(project_id):
 WHERE person_id = {{pid}}
 """)
 
+# Queries to create tables in associated sandbox with rows that will be removed per cleaning rule
```
Comment (ksdkalluri) [inline data_steward/retraction/retract_deactivated_pids.py:36]: These queries are not being used yet, these should belong in a PR where all the queries are being updated.

**Review by ksdkalluri — CHANGES_REQUESTED** [review]

File: data_steward/retraction/retract_deactivated_pids.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/retraction/retract_deactivated_pids.py:36]: I need them for the next ticket, like the util functions. I think they can be moved to a new PR for the main ticket DC-963. What are your thoughts?

Comment (ksdkalluri) [reply data_steward/retraction/retract_deactivated_pids.py:36]: yes, moving this query to that PR will help, and also I think the query will be changed when you are working on it.

**Review by ksdkalluri — APPROVED** [review]
Looks good!

**Pull request merged**